daemon/device-plugin/deviceplugin.go

```diff
@@ -311,10 +265,11 @@ func (nf *nfResources) GetDevicePluginOptions(ctx context.Context, empty *plugin
 	}, nil
 }
 
-func NewGrpcPlugin() *nfResources {
+func NewDevicePlugin(dh DeviceHandler) *nfResources {
```

Comment:
Would it make sense to not get deviceHandler as an argument?

Reply:
Sorry what do you mean? We want to pass the device handler into the device plugin.

Reply:
We agreed to fix this along with main_test.go fixes.